### PR TITLE
K8S Support: Add helm-chart

### PR DESCRIPTION
**Pull request by tibkiss**
Run MarketStore as StatefulSet in Kubernetes. 
Tested with Helm 3.2.0-rc1.

### CODE REVIEW

**Comment by tibkiss** [comment]
Tested locally with an on-prem K8S (v1.14.1):
```
$ helm3 install --namespace mkts mkts helm-chart/
NAME: mkts
LAST DEPLOYED: Fri May  1 09:22:20 2020
NAMESPACE: mkts
STATUS: deployed
REVISION: 1
TEST SUITE: None
$ k get all
NAME                     READY   STATUS    RESTARTS   AGE
pod/mkts-marketstore-0   0/1     Running   0          19s

NAME                       TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
service/mkts-marketstore   ClusterIP   10.233.28.227   <none>        5993/TCP   19s

NAME                                READY   AGE
statefulset.apps/mkts-marketstore   0/1     19s
$ k port-forward svc/mkts-marketstore 5993:5993 &
Forwarding from 127.0.0.1:5993 -> 5993
Forwarding from [::1]:5993 -> 5993
$ curl --data-raw '{"jsonrpc":"2.0", "method":"DataService.ListSymbols", "id":1, "params": {"parameters": {}}}' -H 'Content-Type: application/json' http://localhost:5993/rpc
{"jsonrpc":"2.0","result":{"Results":null},"id":1}
```

File: helm-chart/values.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  # tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+mktsConfig: |-
+  root_directory: /data
+  listen_port: 5993
+
```
Comment (dakimura) [inline helm-chart/values.yaml:14]:
```suggestion
    timezone: UTC
    disable_variable_compression: true
    
```
I would like to add 2 more default configs

File: helm-chart/templates/service.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "marketstore.fullname" . }}
+  labels:
+    {{- include "marketstore.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
```
Comment (dakimura) [inline helm-chart/templates/service.yaml:9]: GRPC port (5995) will be needed here in the near future 💪

File: helm-chart/templates/ingress.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{{- if .Values.ingress.enabled -}}
```
Comment (dakimura) [inline helm-chart/templates/ingress.yaml:1]: I think Ingress is not necessary for now, but I guess this file is made for just in case, correct?

Comment (tibkiss) [reply helm-chart/templates/ingress.yaml:1]: not absolutely necessary, but makes the solution more useful

File: helm-chart/templates/service.yaml (hunk 2 above)
Comment (tibkiss) [reply helm-chart/templates/service.yaml:9]: I'll add this soon!

File: helm-chart/values.yaml (hunk 1 above)
Comment (tibkiss) [reply helm-chart/values.yaml:14]: thanks, will add it today!

**Pull request merged**